.github/workflows/build-docker.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build and publish docker container
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  publish_container:
+    runs-on: ubuntu-latest
+
+    env:
+      CI: true
+      NODE_OPTIONS: --max-old-space-size=4096
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Docker Login
+        run: docker login --username ${{ secrets.DOCKER_HUB_USERNAME }} --password ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
```

Comment:
Is this secure ?
There is this: https://github.com/docker/login-action

Reply:
@iain-b I'll try that other action.

Reply:
If we were logging into and pushing to multiple registries it would make sense to use that. In this case it's just one so something simple is fine IMO.

[GitHub actions automatically redacts from the logs](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets) so there is no danger of them leaking.